contacts/tests.py

```diff
@@ -1,3 +1,125 @@
-# from django.test import TestCase
+import pytest
+from users.models import User, Seeker
+from apartments.models import Apartment, City
+from .models import Connection
+from django.db import transaction, IntegrityError
 
-# Create your tests here.
+
+@pytest.fixture
+def new_city():
+    city = City(cityName='Tel Aviv')
+    city.save()
+    return city
+
+
+@pytest.fixture
+def fixture_make_seeker(new_city):
```

Comment:
Please refrain from mentioning "fixture" in the name, as it is already declared in the decorator. 
a function's name is important and has a huge role in readability, so always pay attention to it.

Reply:
I removed the 'fixture' part and called the function 'make_seeker'